libpromises/processes_select.c

```diff
@@ -181,7 +181,13 @@ static bool SelectProcess(const char *procentry,
 
     for (rp = a->owner; rp != NULL; rp = rp->next)
     {
-        if (SelectProcRegexMatch("USER", "UID", RlistScalarValue(rp), true, names, column))
+        if (rp->val.type == RVAL_TYPE_FNCALL)
+        {
+            Log(LOG_LEVEL_VERBOSE,
+                "Function call '%s' in process_select body was never resolved, skipping",
```

Comment:
So the log message suggests it only appears once, in the last iteration. Did you check how many times it actually appears? If it's multiple times, maybe it should just say *was not resolved*?

Reply:
Fixed.